modules/build/src/main/scala/scala/build/Bloop.scala

```diff
@@ -37,8 +38,10 @@ object Bloop {
   ): Either[Throwable, Boolean] =
     try {
       logger.debug("Listing BSP build targets")
-      val results = buildServer.workspaceBuildTargets()
-        .get(buildTargetsTimeout.length, buildTargetsTimeout.unit)
+      val results = retry()(logger) {
```

Comment:
Och, this can also fail?

Reply:
yup, thus the `ExecutionException` fail 😞